[2022-12-28T04:52:22.842+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T04:52:20.805890+00:00 [queued]>
[2022-12-28T04:52:22.858+0000] {taskinstance.py:1087} INFO - Dependencies all met for <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T04:52:20.805890+00:00 [queued]>
[2022-12-28T04:52:22.858+0000] {taskinstance.py:1283} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T04:52:22.859+0000] {taskinstance.py:1284} INFO - Starting attempt 1 of 1
[2022-12-28T04:52:22.859+0000] {taskinstance.py:1285} INFO - 
--------------------------------------------------------------------------------
[2022-12-28T04:52:23.081+0000] {taskinstance.py:1304} INFO - Executing <Task(PostgresOperator): create_table> on 2022-12-28 04:52:20.805890+00:00
[2022-12-28T04:52:23.088+0000] {standard_task_runner.py:55} INFO - Started process 842 to run task
[2022-12-28T04:52:23.091+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'new_table_and_data_import_dag_v1', 'create_table', 'manual__2022-12-28T04:52:20.805890+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/new_table_and_data_import_dag.py', '--cfg-path', '/tmp/tmpjg7x0d5s']
[2022-12-28T04:52:23.093+0000] {standard_task_runner.py:83} INFO - Job 8: Subtask create_table
[2022-12-28T04:52:23.264+0000] {task_command.py:389} INFO - Running <TaskInstance: new_table_and_data_import_dag_v1.create_table manual__2022-12-28T04:52:20.805890+00:00 [running]> on host 016cb28b89a1
[2022-12-28T04:52:23.515+0000] {taskinstance.py:1513} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=new_table_and_data_import_dag_v1
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2022-12-28T04:52:20.805890+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-12-28T04:52:20.805890+00:00
[2022-12-28T04:52:23.516+0000] {sql.py:253} INFO - Executing: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        
[2022-12-28T04:52:23.525+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-28T04:52:23.775+0000] {base.py:73} INFO - Using connection ID 'financial_database' for task execution.
[2022-12-28T04:52:23.789+0000] {sql.py:364} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS test_table (
                ticker varchar(3),
                year integer,
                quarter integer,
                invest_cost integer,
                from_invest integer,
                from_financial integer,
                from_sale integer,
                free_cash_flow double precision,
                PRIMARY KEY (ticker, year, quarter),
                FOREIGN KEY (ticker) REFERENCES listing_companies (ticker)
            );
        , parameters: None
[2022-12-28T04:52:23.803+0000] {taskinstance.py:1327} INFO - Marking task as SUCCESS. dag_id=new_table_and_data_import_dag_v1, task_id=create_table, execution_date=20221228T045220, start_date=20221228T045222, end_date=20221228T045223
[2022-12-28T04:52:23.864+0000] {local_task_job.py:159} INFO - Task exited with return code 0
[2022-12-28T04:52:23.927+0000] {taskinstance.py:2582} INFO - 1 downstream tasks scheduled from follow-on schedule check
